multinet/swagger/set_metadata.yaml

```diff
@@ -0,0 +1,35 @@
+Set the metadata of a table.
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: metadata
+    in: body
+    description: The metadata to set (overwrites existing data)
+    required: true
+    schema:
+      type: object
+      example:
+        columns:
+          - key: test
+            type: label
+
+          - key: length
+            type: number
+
+
```

Comment:
```suggestion
```

Reply:
Not sure why this is showing removal of only one blank line. My suggestion is to reduce the three blank lines between these sections to just one.